### PR TITLE
add support to config SuccessfulJobsHistoryLimit

### PR DESCRIPTION
**Pull request by peacecoder**
## Which problem is this PR solving?
- please refer to the jaegertracing/jaeger-operator#621 for more details.

## Short description of the changes
- I add new config about setting the number of successful job will will be kept in history, which supports for esRollover, esIndexCleaner, dependencies.

### CODE REVIEW

File: content/docs/next-release/operator.md [hunk 1]
```diff
@@ -389,6 +389,7 @@ storage:
     numberOfDays: 7                               // number of days to wait before deleting a record
     schedule: "55 23 * * *"                       // cron expression for it to run
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
+    successfulJobsHistoryLimit: 0                 // number of successful job will will be kept in history, support for esRollover, esIndexCleaner, dependencies.
```
Comment (pavolloffay) [inline content/docs/next-release/operator.md]: Could you please add this also to the last releases if the feature was available there too?

Comment (peacecoder) [reply content/docs/next-release/operator.md]: after merge https://github.com/jaegertracing/jaeger-operator/pull/621, I will add it to the latest release. Thanks

Comment (peacecoder) [reply content/docs/next-release/operator.md]: Hi, https://github.com/jaegertracing/jaeger-operator/pull/621 was merged yesterday

Comment (pavolloffay) [reply content/docs/next-release/operator.md]: The comment does not seem right:  // number of successful job will will be kept in history,` It should rather be ` // number of successful jobs that will be kept in history,`

**Comment by yurishkuro** [comment]
@pavolloffay @peacecoder what is the status of this PR?

**Comment by jpkrohling** [comment]
I'm closing this PR, as there's no feedback even after a status request a couple of months ago.

**Pull request closed without merging**